src/emulator/constants.ts

```diff
@@ -124,4 +128,8 @@ export class Constants {
     const u = url.parse(normalized);
     return u.hostname || DEFAULT_HOST;
   }
+
+  static isFakeProject(projectId?: string): boolean {
```

Comment:
nit: consider naming it `isDemoProject` to avoid misunderstanding as "projects that don't exist"